src/metabase/api/fingerprint.clj

```diff
@@ -0,0 +1,134 @@
+(ns metabase.api.fingerprint
+  (:require [compojure.core :refer [GET]]
+            [metabase.api.common :as api]
+            [metabase.fingerprinting.core :as f]
+            [metabase.models
+             [card :refer [Card]]
+             [field :refer [Field]]
+             [metric :refer [Metric]]
+             [segment :refer [Segment]]
+             [table :refer [Table]]]
+            [schema.core :as s]))
+
+;; See metabase.fingerprinting.core/fingerprint for description of these settings.
+(def ^:private ^:const MaxQueryCost
```

Comment:
generally I don't think you should make schemas `^:const` otherwise they get copied into every place that references them. Some schemas can get pretty huge. Also some of the ones that use predicate functions wouldn't work as `^:const` and would make the compiler complain